packages/bezier-react/src/components/AlphaStatusBadge/StatusBadge.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { type CSSProperties, forwardRef, memo } from 'react'
+
+import { MoonFilledIcon } from '@channel.io/bezier-icons'
+import classNames from 'classnames'
+
+import { cssVar } from '~/src/utils/style'
+
+import { Icon } from '~/src/components/Icon'
+
+import { type StatusBadgeProps } from './StatusBadge.types'
+
+import styles from './StatusBadge.module.scss'
+
+/**
+ * `StatusBadge` is a component to indicate user status.
+ * @example
+ *
+ * ```tsx
+ * <StatusBadge
+ *   size="m"
+ *   online
+ *   doNotDisturb
+ * />
+ * ```
+ */
+export const StatusBadge = memo(
+  forwardRef<HTMLDivElement, StatusBadgeProps>(function StatusBadge(
+    {
+      size = 'm',
+      online = false,
+      doNotDisturb = false,
+      style,
+      className,
+      ...rest
+    },
+    forwardedRef
+  ) {
+    const withIcon = doNotDisturb
+    const iconColor = online
+      ? 'bgtxt-green-normal'
+      : doNotDisturb
+        ? 'bgtxt-yellow-normal'
+        : 'bg-black-dark'
+    const backgroundColor = withIcon ? 'bg-white-high' : iconColor
+
+    return (
+      <div
+        ref={forwardedRef}
+        style={
+          {
+            '--b-status-bg-color': cssVar(backgroundColor),
+            ...style,
+          } as CSSProperties
+        }
+        className={classNames(styles.Status, styles[`size-${size}`], className)}
```

Comment:
가장 바깥에 있는 엘리먼트의 className은 컴포넌트 이름과 동일하게 맞추고 있습니다 
```suggestion
        className={classNames(styles.StatusBadge, styles[`size-${size}`], className)}
```